terraform/modules/logging/alarms/admin-manager-password-reset/alarms.tf

```diff
@@ -6,9 +6,9 @@ resource "aws_cloudwatch_metric_alarm" "admin_manager_password_reset" {
   namespace   = "DM-admin-manager-password-reset"
   metric_name = "${var.environment}-admin-manager-password-reset"
 
-  // For for every 60 seconds
+  // For for every 300 seconds
   evaluation_periods = "1"
-  period             = "60"
+  period             = "300"
```

Comment:
I agree with the commit message - as discussed elsewhere, if the attacker is sophisticated enough to piggyback on a valid reset, 1 minute vs 5 minutes is unlikely to make a difference. However we should make a note to review this as a key alert if/when we get set up with Splunk.

Reply:
I'll add the comment about Splunk to the tech debt card

Reply:
tech debt card added, resolving